src/GridFS/CollectionWrapper.php

```diff
@@ -150,9 +150,6 @@ public function findChunksByFileId(mixed $id, int $fromChunk = 0)
      */
     public function findFileByFilenameAndRevision(string $filename, int $revision): ?object
     {
-        $filename = $filename;
-        $revision = $revision;
```

Comment:
I thought I had already removed that.

Reply:
Noted that this remained when https://github.com/mongodb/mongo-php-library/commit/1b29a6d71e5511ea46b596461442ca94069bdf42 introduced type hints on the method.